### ISSUE TITLE
Missing dot (.) in pre-release versioning

### ISSUE THREAD

**Issue by cmergenthaler** [christian-draeger/bump-semantic-version#8]
As in https://semver.org/ stated a pre-release (alpha, beta, rc) is appended by a series of dot-separated identifiers:
```
<pre-release> ::= <dot-separated pre-release identifiers>
<dot-separated build identifiers> ::= <build identifier>
                                    | <build identifier> "." <dot-separated build identifiers>
```
Currently pre-releases are not separated with dot-separated identifiers. So instead of `2.11.7-rc1` it should be `2.11.7-rc.1`

**Comment by christian-draeger** [comment]
Thx for pointing this out.
I will have a look at it

**Comment by valleywood** [comment]
Any chance the dot will be in next release (1.0.4)?

**Comment by NatoNathan** [comment]
Hi, I have just open a PR that looks to support a dot #13

**Comment by christian-draeger** [comment]
just released within version 1.1.0
